plugins.yml

```diff
@@ -31,14 +31,12 @@ plugins:
             'shop_administration/manage_products.md': 'pim/products.md'
             'shop_administration/product_categories.md': 'pim/work_with_product_categories.md'
             'shop_administration/catalogs.md': 'pim/work_with_catalogs.md'
-            'shop_administration/cockpit.md': 'shop_administration/discover_dashboard.md'
+            'shop_administration/cockpit.md': 'cdp/customer_portal.md'
```

Comment:
CDP usually refers to "Customer Data Platform" and have nothing in common with customer portals/corporate accounts. Are you sure it's the right place for "Company self-registration", "Customer Portal account" articles ?

Reply:
I see your pint now. Yes, I will move this content to its own category of customer management.